data/STRchive-loci.json

```diff
@@ -2267,8 +2267,8 @@
   "benign_min": 6.0,
   "benign_max": 26.0,
   "intermediate_min": 27.0,
-  "intermediate_max": 39.0,
-  "pathogenic_min": 40.0,
+  "intermediate_max": 35.0,
+  "pathogenic_min": 36.0,
```

Comment:
Could you please add a comment in the details section of HTT saying that alleles of 40+ are typically fully penetrant, with a citation [@genereviews:NBK1305]? It is implied, but would be good to make it explicit.